website/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
I think we can remove this file too, WDYT?

Reply:
I agree with that, we have a licence at the root level